### DISCUSSION TITLE
additionalReviewers treated as groups instead of users in GitLab

### DISCUSSION THREAD

**Discussion by tymofiismyrnov-mojam** [renovatebot/renovate#33288] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I am experiencing an issue with the additionalReviewers configuration in Renovate when using GitLab. It appears that each user in the list is being treated as a group, resulting in API calls to the GitLab API that fail with a 404 error.

**Configuration**:
Here is an excerpt from my **renovate.json5**
`{"additionalReviewers": ["tymofii.smyrnov"]}`

### Logs (if relevant)

<details><summary>Logs</summary>

```
DEBUG: Additional reviewers: "tymofii.smyrnov" (repository={my_repo}, branch={my_branch})
DEBUG: Adding reviewers 'tymofii.smyrnov' to #13 (repository={my_repo}, branch={my_branch})
DEBUG: getMR(13) (repository={my_repo}, branch={my_branch})
DEBUG: GET https://{my_gitlab}/api/v4/groups/tymofii.smyrnov/members = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=42) (repository={my_repo}, branch={my_branch})
DEBUG: GitLab API 404 (repository={my_repo}, branch={my_branch})
       "url": "https://{my_gitlab}/api/v4/groups/tymofii.smyrnov/members"
DEBUG: Aggregate error is thrown (repository={my_repo}, branch={my_branch})
       "err": {
         "name": "AggregateError",
         "message": "\n    HTTPError: Response code 404 (Not Found)\n        at Request.<anonymous> (/usr/local/renovate/node_modules/.pnpm/got@11.8.6/node_modules/got/dist/source/as-promise/index.js:118:42)\n        at processTicksAndRejections (node:internal/process/task_queues:95:5)",
         "stack": "AggregateError: \n    HTTPError: Response code 404 (Not Found)\n        at Request.<anonymous> (/usr/local/renovate/node_modules/.pnpm/got@11.8.6/node_modules/got/dist/source/as-promise/index.js:118:42)\n        at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at next (/usr/local/renovate/node_modules/.pnpm/p-map@4.0.0/node_modules/p-map/index.js:43:14)\n    at /usr/local/renovate/node_modules/.pnpm/p-map@4.0.0/node_modules/p-map/index.js:59:6\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"

```

</details>

**Comment by RahulGautamSingh** [comment]
First we try to find the user id by sending a GET req to the `/users?username=<name>` endpoint. If it fails then we move to the next code which is to treat user a group. But, since your username is valid..I don't get how it is failing.

Can you reproduce it in a gitlab repo and link it? I will try it and create a issue.
